jsreport-linux-appservice-docker/README.md

```diff
@@ -0,0 +1,16 @@
+# Provision jsreport service in Azure
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2Fjsreport-linux-appservice-docker%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fjsreport-linux-appservice-docker%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template allows you to deploy jsreport, a neat and elegant library / service for drawing PDF Reports using HTML and handlebars Templates. You can pick from a variety of recipes aka Algorithms to generate PDF’s using engines like PhantomPDF, Electron,Chromium,WkhtmltoPDF etc.., This Template will create a Linux App Service + StorageAccount, Pull the Jsreports 2.1.0-full version build from Docker Registry and Configure Authentication and Embed All the settings required to connect to a Storage Account in order to Persist the Templates.
+
+The default credentials to log into jsreport service will be 'username:admin,password:password'
```

Comment:
the creds need to be passed in by the user

Reply:
This is now removed from README